engine/schema/src/com/cloud/vm/UserVmDetailVO.java

```diff
@@ -80,4 +80,8 @@ public boolean isDisplay() {
         return display;
     }
 
+    public void setValue(String value) {
```

Comment:
I removed it as with new changes in `VMSnapshotManagerImpl`, it is no longer needed, thanks!